scripts/azure-price-fetch/types.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+import "gomodules.xyz/azure-retail-prices-sdk-for-go/sdk"
+
+type VirtualMachineInfo struct {
+	Name        string
+	OwningVMSS  string
+	MachineType string
+}
+
+type VmMap struct {
+	RegionMap map[string]map[string]VirtualMachineInfo
+}
+
+type PriceMap struct {
+	RegionMap map[string]map[string]sdk.ResourceSKU
+}
```

Comment:
IMO having a dedicated `types` package is a bit of an antipattern from other languages and makes it harder to follow the code. I'd have these at the top of `azureClientWrapper.go` so one doesn't need to jump between multiple files. Often times I'll see a `types.go` like file for generated code :)

Reply:
moved to the other file